# HG changeset patch
# User Rik <rik@octave.org>
# Date 1383600004 28800
#      Mon Nov 04 13:20:04 2013 -0800
# Node ID e432697a0621425bb5a40e55529d86763f15e05c
# Parent  60b63193a33f2fa5b1d58f4dc60ecb3511406ccb
octave.cc: Set LC_TIME to C locale so that dates always compute correctly.

* libinterp/octave.cc: Set LC_TIME to C locale so that dates always compute correctly.

diff --git a/libinterp/octave.cc b/libinterp/octave.cc
--- a/libinterp/octave.cc
+++ b/libinterp/octave.cc
@@ -702,16 +702,17 @@ octave_process_command_line (int argc, c
 // EMBEDDED is declared int instead of bool because this function is
 // declared extern "C".
 
 void
 octave_initialize_interpreter (int argc, char **argv, int embedded)
 {
   // Matlab uses "C" locale for LC_NUMERIC class regardless of local setting
   setlocale (LC_NUMERIC, "C");
+  setlocale (LC_TIME, "C");
 
   octave_embedded = embedded;
 
   octave_env::set_program_name (argv[0]);
 
   octave_program_invocation_name = octave_env::get_program_invocation_name ();
   octave_program_name = octave_env::get_program_name ();
 
